modules/sdk-core/src/bitgo/lightning/iLightning.ts

```diff
@@ -153,6 +159,25 @@ export const GetInvoicesResponse = t.array(InvoiceInfo);
 // eslint-disable-next-line no-redeclare
 export type GetInvoicesResponse = t.TypeOf<typeof GetInvoicesResponse>;
 
+const PaymentInfo = t.strict({
+  paymentHash: t.string,
+  walletId: t.string,
+  status: t.union([t.literal('in_flight'), t.literal('settled'), t.literal('failed')]),
+  amount: t.number,
+  invoice: t.string,
+  sendQueueId: t.string,
+  failureReason: t.string,
+  fee: t.number,
+  feeLimit: t.number,
+  paymentPreimage: t.string,
+  destination: t.string,
+});
+
+export const GetPaymentsResponse = t.array(PaymentInfo);
+
+// eslint-disable-next-line no-redeclare
```

Comment:
Might be nice to declare this rule for the whole file rather than every time a type is declared.